components/Table/TableContainer.jsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
+import HardwareKeyboardArrowRight from 'material-ui/svg-icons/hardware/keyboard-arrow-right';
 import styles from './TableContainer.css';
 
 const TableContainer = ({ title, style, children }) => (
   <div className={styles.container} style={{ ...style }}>
     <div className={styles.heroesContainer}>
-      <div className={styles.tableHeading}>{title}</div>
+      <div className={styles.tableHeading}>
+        <HardwareKeyboardArrowRight style={{ verticalAlign: 'top', opacity: '.6', height: 26 }} />
+        {title}
+      </div>
```

Comment:
Agreed, it does seem to add something nice with something in front of the title. You seem to have a good eye for design so please feel free to mess around with stuff or disagree with me since I am not a designer nor do I have a natural talent for it.